tests/conftest.py

```diff
@@ -392,3 +370,20 @@ def random_dt(monkeypatch):
     """Make ``datetime.datetime.utcnow()`` return an arbitrary date."""
     with mocked_datetime(monkeypatch) as datetime:
         yield datetime.utcnow()
+
+
+@pytest.fixture(scope="function")
+def setup_pickleddb_database():
+    """Configure the database"""
+    with OrionState() as cfg:
+        yield cfg
+
+
+@pytest.fixture(scope="function")
+def storage(setup_pickleddb_database):
+    yield setup_pickleddb_database.storage
+
+
+@pytest.fixture(scope="function")
+def orionstate(setup_pickleddb_database):
```

Comment:
Maybe the use of setup_pickleddb_database should be renamed to orionstate instead. There are many instances of `cfg = setup_pickleddb_database`.

Reply:
done